python-threatexchange/threatexchange/cli/dataset_cmd.py

```diff
@@ -96,58 +110,67 @@ def init_argparse(cls, settings: CLISettings, ap: ArgumentParser) -> None:
         )
         ap.add_argument(
             "--signals-only",
-            "-i",
+            "-S",
             action="store_true",
             help="[-P] only print signals",
         )
         ap.add_argument(
-            "--limit",
-            "-l",
+            "--csv",
             action="store_true",
-            help="[-P] only print this many records",
+            help="[-P] store in csv format",
```

Comment:
store -> output
The word store really confused me and I started looking for a filename arg/default